### PR TITLE
Force bitrate to minimum 6000kbps

### PR DESCRIPTION
**Pull request by lsegal**

### CODE REVIEW

**Comment by aravindh-rq** [comment]
@lsegal  can we use/send the props from dart to native side, so that it will be customised based on requirement in a simple way right.

let me know your suggestion on this.

**Comment by lsegal** [comment]
This is just changing default values. I believe you can still customize by sending a bitrate value.

File: ios/Classes/FlutterRTMPStreaming.swift [hunk 1]
```diff
@@ -187,34 +187,34 @@ public class FlutterRTMPStreaming : NSObject {
 
 
 class MyRTMPStreamQoSDelagate: RTMPStreamDelegate {
-    let minBitrate: UInt32 = 300 * 1024
-    let maxBitrate: UInt32 = 2500 * 1024
+    let minBitrate: UInt32 = 6000 * 1024
```
Comment (lsegal) [inline ios/Classes/FlutterRTMPStreaming.swift:190]: You can adjust this to a lower value (like `1000 * 1024`) if you want it to be further customizable. The increasing of the maxBitrate below is the important part to allow a 6000kbps rate.

**Comment by aravindh-rq** [comment]
Thanks @lsegal will merge it to the master.

**Pull request merged**